src/routes/appointments.js

```diff
@@ -1,12 +1,23 @@
 const router = require('express').Router();
 const Joi = require('joi');
-const { getAppointmentsByDate } = require('../controllers/appointments');
 const wrapper = require('../middleware/wrapper');
 const validator = require('../middleware/validator');
 
+const { getAppointmentsByDate } = require('../controllers/appointments');
```

Comment:
U don't need copy requirement, just add in the whole of the first requirement.

Reply:
done